benchmark/text-generation-inference/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.7'
 
 services:
   tgi-1:
-    image: neuronx-tgi:latest
+    image: ghcr.io/huggingface/neuronx-tgi:latest
```

Comment:
Actually, neuronx-tgi is the name of the image when it is built locally, for instance by typing `make neuronx-tgi`.

Reply:
So you can make it clearer in the README, but I'd rather not change the compose file.